site/package.json

```
@@ -3,39 +3,39 @@
   "version": "0.0.0-private",
   "private": true,
   "dependencies": {
-    "@babel/core": "^7.6.0",
-    "@babel/plugin-proposal-class-properties": "^7.5.0",
-    "@babel/plugin-proposal-numeric-separator": "^7.2.0",
-    "@babel/plugin-proposal-object-rest-spread": "^7.5.4",
-    "@babel/preset-react": "^7.0.0",
-    "@babel/preset-typescript": "^7.6.0",
+    "@babel/core": "^7.13.10",
+    "@babel/plugin-proposal-class-properties": "^7.13.0",
+    "@babel/plugin-proposal-numeric-separator": "^7.12.13",
+    "@babel/plugin-proposal-object-rest-spread": "^7.13.0",
+    "@babel/preset-react": "^7.12.13",
+    "@babel/preset-typescript": "^7.13.0",
     "@kaizen/component-library": "*",
     "@kaizen/design-tokens": "*",
-    "@kaizen/hosted-assets": "^1.0.1",
+    "@kaizen/hosted-assets": "^1.1.0",
     "@mdx-js/mdx": "^1.4.5",
     "@mdx-js/react": "^1.4.5",
     "classnames": "^2.2.6",
     "color": "^3.1.2",
-    "gatsby": "^2.24.74",
-    "gatsby-plugin-force-trailing-slashes": "^1.0.4",
-    "gatsby-plugin-google-tagmanager": "^2.2.1",
-    "gatsby-plugin-mdx": "^1.0.43",
-    "gatsby-plugin-react-helmet": "^3.1.8",
-    "gatsby-plugin-sass": "^2.1.15",
-    "gatsby-plugin-svg-sprite": "^2.0.1",
-    "gatsby-source-filesystem": "^3.0.0",
-    "gatsby-transformer-remark": "^2.6.24",
+    "gatsby": "2.24.74",
+    "gatsby-plugin-force-trailing-slashes": "^1.0.5",
```

Did you want to remove the `^` here?
```suggestion
    "gatsby-plugin-force-trailing-slashes": "1.0.5",
```

(You'd need to re-run yarn after committing that)

---

Fixed now, thanks :D